style/style.css

```diff
@@ -28,12 +34,70 @@ ul {
   list-style: none;
 }
 
+.btn {
```

Comment:
- [ ] OPTIONAL: All the `See Project' buttons should have been similar `btn`, '`btn2', `btn3' and `btn4`. In addition, you should have styling for hover, enabled, and disabled states of the button.